src/plugins/intel_cpu/src/nodes/mkldnn_eltwise_node.cpp

```diff
@@ -81,8 +99,11 @@ template <cpu_isa_t isa>
 struct jit_uni_eltwise_generic : public MKLDNNPlugin::jit_uni_eltwise_kernel, public jit_generator {
     DECLARE_CPU_JIT_AUX_FUNCTIONS(jit_uni_eltwise_generic)
 
-    explicit jit_uni_eltwise_generic(const jit_eltwise_params& jep, MKLDNNEltwiseNode& eltwiseNode) :
-        jit_uni_eltwise_kernel(jep, eltwiseNode), jit_generator() {}
+    explicit jit_uni_eltwise_generic(jit_eltwise_params jep,
```

Comment:
Here we use pretty common idiom, where we take a parameter by value and then we pass it further as an rvalue reference, so it allows us to get some benefits from the move semantics.  In general case this approach may be criticized since not all classes have cheap move constructors, but  in this particular case moving jit_eltwise_params does not seems to be expensive. On the other hand in case of passing lvalue reference we will have pretty the same overheads as with conventional `const &` signature.

Reply:
Considering coverity issues with this approach, the constructor signature was changed to const reference usage.